RPS.html

```diff
@@ -67,27 +67,61 @@
       }
     }
 
+    let playerScore = 0;
+    let computerScore = 0;
```

Comment:
I'd rather place those in the beginning, where the `const rock` etc is. It'll be much easier to find it.